### PR TITLE
Add Lobby class

### PR DESCRIPTION
**Pull request by RileyChampion**

### CODE REVIEW

File: src/main/java/com/example/TwoRoomsBoom/models/Lobby.java [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+@Builder
+@AllArgsConstructor
+@NoArgsConstructor
+public class Lobby {
+    private String lobbyId;
+    private List<Role> rolesAdded;
+    private List<Player> players;
+    private Set<Player> readyPlayers;
+    private Player host;
+    private String settings;
```
Comment (Ji-Justin) [inline src/main/java/com/example/TwoRoomsBoom/models/Lobby.java:24]: Just thinking about this now, how should we setup settings? Should we make an array containing a boolean for bury cards and ints for amount of rounds and time?

Comment (RileyChampion) [reply src/main/java/com/example/TwoRoomsBoom/models/Lobby.java:24]: Probably I think that maybe a Settings class would be helpful potentially?

Comment (RileyChampion) [reply src/main/java/com/example/TwoRoomsBoom/models/Lobby.java:24]: Like as I was making it I was wondering if there should be a dedicated class to hold the data values of the settings for that lobby

Comment (Ji-Justin) [reply src/main/java/com/example/TwoRoomsBoom/models/Lobby.java:24]: You're right, a Settings class would probably be ideal, in the case that we have more things to add later down the line

**Pull request merged**